### PR TITLE
Include edits by Antonella Iecle

### PR DESCRIPTION
**Pull request by bonzini**

### CODE REVIEW

**Comment by bonzini** [comment]
Not sure how I can squash these edits from GitHub, so do that when merging the pull request if everything is okay :)

**Comment by grivalta** [comment]
Hi, many thanks to you and Antonella Iecle for this edit. Soon i give it a more deep reading to finalize the merge...

File: it_IT/ODF-community.md [hunk 1]
```diff
@@ -23,18 +23,18 @@ Contenuti
 ===========================================================================
 * [Panoramica](#panoramica)
   * [Che cos'è una decisione aperta?](#what-is-an-open-decision)
-  * [Le decisioni aperte sono prese secondo i principi Open Source](#open-decisions-are-made-using-open-source-principles)
```
Comment (grivalta) [inline it_IT/ODF-community.md:26]: I do not know at this point the real meaning of the English assertion. I thought about the decision-making process. That is, "Open decisions are taken through an open approach" and not that their content speaks about this approach ...
But mind, I'm a base english speaker :)

File: it_IT/ODF-community.md [hunk 2]
```diff
@@ -23,18 +23,18 @@ Contenuti
...
-  * [In che modo i principi Open Source conducono a decisioni migliori](#how-open-source-principles-lead-to-better-decisions)
-  * [Non si può piacere a tutti](#you-cant-please-everyone)
-* [Sistema decisionale aperto](#open-decision-framework-1)
-  * [Fase o attività: Concettualizzare, Definire, Ideare](#phase-or-activity-concept-define-ideate)
-  * [Fase o attività: Pianificare, Ricercare](#phase-or-activity-plan-research)
-  * [Fase o attività: Concepire, Sviluppare, Verificare](#phase-or-activity-design-develop-test)
-  * [Fase o attività: Avviare, Mettere in campo, Chiudere](#phase-or-activity-launch-deploy-close)
+  * [Le decisioni aperte rispecchiano i principi Open Source](#open-decisions-are-made-using-open-source-principles)
```
Comment (grivalta) [inline it_IT/ODF-community.md]: See https://github.com/grivalta/open-decision-framework/pull/2/commits/9e33f2bc51d9a60c942f2dd286dda151e9eddafe#r243731283

File: it_IT/ODF-community.md [hunk 3]
```diff
@@ -156,7 +156,7 @@ _page 7 of 15_
 </tr>
 <tr valign="top">
 <td rowspan=3>
-<strong>Agite con trasparenza</strong><br/>
+<strong>Agite con trasparenza <!-- "siate leader trasparenti?" "mostrate la via nella trasparenza?" </strong><br/>
```
Comment (grivalta) [inline it_IT/ODF-community.md:159]: Based on the three points of detail, may be it does not concern only the aspect of good leadership, even if obviously the action is always "give the example". What do you think about the following alternatives?

- "Comportatevi in modo trasparente"
- "Operate in modo trasparente"

File: it_IT/ODF-community.md (hunk 1 above)
Comment (bonzini) [reply it_IT/ODF-community.md:26]: Right, even better would be "Il processo decisionale aspetto rispecchia i principi open source". I liked the use of the active voice, passive sounds really bad in this sentence.

 I will shortly push an update.

File: it_IT/ODF-community.md (hunk 3 above)
Comment (grivalta) [reply it_IT/ODF-community.md:159]: "Gestite con trasparenza" ?

File: it_IT/ODF-community.md (hunk 1 above)
Comment (bonzini) [reply it_IT/ODF-community.md:26]: Done now. I changed it to "Taking decisions according to open source principles", which I think matches the original English text pretty well.

**Comment by grivalta** [comment]
I merged now your PR, squashing all the commits. Next days i try a global review of the text to propose the merge into Red Hat People team repo...

**Pull request merged**